charts/mimir-distributed/templates/nginx/ingress.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.enterprise.enabled -}}
```

Comment:
see comment about gateway, but WDYT about using nginx for both enterprise and OSS?

Reply:
see answer above :)